### ISSUE TITLE
Feature Request: enable field values of colorit for default GraphQL API

### ISSUE THREAD

**Issue by bzin** [presseddigital/colorit#18]
Just notice #14 is about the same thing but that one was closed. It would be great to bring this back and have this on the plugin.

I am currently building an application that uses the GraphQL and was considering using `colorit`. Although this only returns the `hex` value which is not sufficient for what I indent to build.

Maybe consider adding it is a new release?

**Comment by michtio** [comment]
agreed, being able to query for handle or label would be very useful here

**Comment by samhibberd** [comment]
What subfields would you like to see, thinking:

```
colourField {
  label
  handle
  color
  hex
  rgb
  rgba 
  opacity
}
```
